PixelHunter1995/Components/CharachterComponent.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Content;
+using Microsoft.Xna.Framework.Graphics;
+using PixelHunter1995.TilesetLib;
+using System;
+
+namespace PixelHunter1995.Components
+{
+    class CharachterComponent : IComponent, ICharacterComponent
```

Comment:
A minor typo: "Charachter" -> "Character"
Not really an issue though, and maybe intended (ie. to make a note to use different terminology for characters in a string vs characters in the game to avoid confusion anywhere)?

Reply:
done